src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -54,6 +54,7 @@
   --pf-c-context-selector__menu-list--MaxHeight: #{pf-size-prem(200px)};
 
   // Menu item
+  --pf-c-context-selector__menu-list-item--Color: var(--pf-global--Color--dark-100);
```

Comment:
Should `--pf-c-context-selector__menu-list-item--Color` be in the CSS variables section of the docs?

Reply:
@redallen can you confirm?

Reply:
It should be in that section once `react-tokens` takes in the change.